include/pacmod3/pacmod3_nodelet.h

```diff
@@ -141,7 +141,9 @@ class Pacmod3Nl : public nodelet::Nodelet
   ros::Publisher all_system_statuses_pub;
   ros::Publisher global_rpt_pub;
   ros::Publisher global_rpt_2_pub;
```

Comment:
All these publisher variables should be deleted because they are stored into `pub_tx_list` with `std::move`.